openassessment/mobile_assessment/models.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import models
+
+
+class AssessmentTemplate(models.Model):
```

Comment:
Let's please rename the Model, because  AccessmentTemplate sounds like its template of whole ORA assessment, but actually it's template of ORA table template.